Sources/BSON/BSONValue.swift

```diff
@@ -14,6 +14,11 @@ internal protocol BSONValue: Codable {
 
     /// Writes this value's BSON byte representation to the provided ByteBuffer.
     func write(to buffer: inout ByteBuffer)
+
+    /// Initializes a corresponding `BSON` from the provided extendedJSON.
+    /// Currently commented out to avoid "not implemented" errors while implementing the
+    /// initializer one by one on each BSONValue.
+    // internal init(fromExtJSON json: JSON) throws
```

Comment:
should I just delete this for now? and add it back in after all the implementations

Reply:
I think it's fine to just leave it in.